lib/new_utils.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+/**
+ * @param {String} entity
+ * @param {String} name
+ */
+function throwNotSupported(entity, name) {
+    throw new ReferenceError(
+        `The ${entity} (${name}) is not supported by our driver`,
+    );
+}
+
+exports.throwNotSupported = throwNotSupported;
```

Comment:
As suggested, we could export only concrete variants:
`throwNotSupportedFunction`, `throwNotSupportedClass`, etc.